tests/jacobi/include/helpers-jacobi.hpp

```diff
@@ -320,55 +319,69 @@ void read_args(int argc, char **argv){
 
 
 //print matrix : row major, i = line, j = column
-void print_host2D(float * tab) {
-  for (size_t i = 0; i < M; ++i){
-    for(size_t j = 0; j < N; ++j){
-      std::cout << tab[i*N+j] << " " ;
+void print_host2D(float * tab)
```

Comment:
The issue is not to revert but to have this never happened, otherwise the lines may be seen as having written by you in `git annotate`.
This is a good exercise to make progress with Git and your favorite client. :-)
Rewriting the branch by carefully selecting some hunks and removing some changes, and at the end pushing in force when you are happy...
Emacs Magit mode for example have some interesting mode to select and cherrypick some hunks.
I do not know about the Git-mode of latest Visual Studio.